python/py-mediafile/Portfile

```diff
@@ -21,25 +21,23 @@ homepage            https://mediafile.readthedocs.io/
 
 maintainers         {@catap korins.ky:kirill} openmaintainer
 
-checksums           rmd160  9d9a780fe149b6a15a9ca0b50332185aecc0b14e \
-                    sha256  93ccef3fbb7d4554a0e7689d41236cd5686a2f2f17493098622b8344cf83df9a \
-                    size    561866
+checksums           rmd160  e1796e3c1bf38739afc6d5321262d4c6b08138fe \
+                    sha256  929642a17ee5023b9086221ce80cdc2c51d06021a890d4433b0bd5bd32f2b29f \
+                    size    563478
 
+python.pep517       yes
 python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
+                    port:py${python.version}-flit_core \
                     port:py${python.version}-setuptools
 
     depends_lib-append \
                     port:py${python.version}-six \
                     port:py${python.version}-mutagen
 
     test.run        yes
-    test.cmd        ${python.bin}
-    test.args       -m unittest discover
```

Comment:
did upstream swith to useing `pytest` as the testing framework? If not, then this will not work as the default `python.test_framework` is set to `pytest`; if you want to use `unittest` you should specify that instead.

Reply:
@reneeotten yep, they did: https://github.com/beetbox/mediafile/commit/e867bfa9c1cdaa9d128deb8534d366e0b79379c4